backend/api/Areas/Keycloak/Mapping/User/UserMap.cs

```diff
@@ -66,7 +66,7 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.Roles, src => src.Roles)
                 .Inherits<Entity.BaseEntity, Model.Update.BaseModel>();
 
-            config.NewConfig<Model.Update.UserModel, Entity.User>()
```

Comment:
Are you absolutely certain that this mapping is no longer required anywhere?

Reply:
I am certain